documentation/manual/src/main/asciidoc/modules/mongodb.asciidoc

```diff
@@ -9,6 +9,34 @@ The nested nature of documents make it a particularly natural fit for most objec
 This implementation is based upon the MongoDB Java driver.
 The currently supported version is {mongodb-version}.
 
+=== Why should I use Hibernate OGM with MongoDB
+
+It's possible that in your project you have some entities that might benefit from MongoDB
+dynamic schema, but having a schema makes it possible to obtain better performance because
+the datastore can use the information of the schema to apply some optimizations that
+wouldn't be otherwise possible.
+
+JPA already has ways to define constraints and indexes and via Hibernate OGM you can
+use the same annotations for both your relational and not relational needs.
+
+Hibernate OGM cannot make MongoDB transactional but by using the JPA transaction demarcation
+mechanism it can group the operations and flush them to the datastore to
+minimize the number of requests.
+
+Another benefit of using Hibernate OGM with MongoDB is that it will also make it possible
+to use Hibernate Search out of the box. Hibernate Search brings the power of Lucene
+to your project, giving you the ability to run fast google-like searches.
+
+This means that you can query the datastore:
+
+* <<ogm-jpql-query>>
+* using <<ogm-mongodb-queries-native>>
+* Full-text queries using <<ogm-query-using-hibernate-search>>
```

Comment:
This list looks weird and inconsistent. Could we make the wording consistent? I'm especially worried about the first item of the list: relying on the title of the other paragraph does not look like a good idea.